resources/ec2_security_groups.go

```diff
@@ -475,3 +481,9 @@ func fetchEc2SecurityGroupIpPermissionsEgressUserIdGroupPairs(ctx context.Contex
 	res <- securityGroupIpPermissionEgress.UserIdGroupPairs
 	return nil
 }
+
+func resolveSGArn(_ context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	cl := meta.(*client.Client)
+	sg := resource.Item.(types.SecurityGroup)
```

Comment:
Didn't touch that part of the code for a while but is a cast check needed here?

Reply:
yes, its advised, although unlikely to happen, its better than a panic.

Reply:
@roneli won't it still panic since it's not in the `sg, ok` syntax?

Reply:
sorry, maybe my wording was incorrect. I suggest we do the `sg, ok` assertion syntax, since we prefer an error over a panic.

Reply:
What error should I return?